packages/address/src/MultisigAddress.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  bs58M,
+  bs58N,
+  bs58Version,
+  bs58MultisigPublicKey,
+  bs58NetType,
+  byteToNetType,
+  byteToKeyType,
+  sortAddresses,
+  bs58KeyType,
+} from './utils'
+import { sha256 } from 'multiformats/hashes/sha2'
+import { MULTISIG_KEY_TYPE } from './KeyTypes'
+import { NetType, MAINNET} from './NetTypes'
+import Address from './Address'
+
+export class MultisigAddress extends Address {
+  public M!: number
```

Comment:
Are `M` and `N` standard terms? I am not familiar with multisig implementations, but if possible can we provide a more descriptive name for the variables?

Reply:
If I understand this properly, it's also always two signatures? Is there possibility for more than two?

Reply:
> Are `M` and `N` standard terms? I am not familiar with multisig implementations, but if possible can we provide a more descriptive name for the variables?

The `M` and `N` are taken from the reference implementation, but I agree they're nondescript and can be named better - will update.

> If I understand this properly, it's also always two signatures? Is there possibility for more than two?

I'm no expert but my understanding is that there is no theoretical limit to the number of signers (other than physical payload limitations on the blockchain side that would prevent a large number of accounts/signers).

Reply:
Yes, "m-of-n" is a de facto standard term in multisig land. Example: https://en.bitcoin.it/wiki/Multi-signature

Reply:
Yes, there's no theoretical limit on N and M is constrained to 0 < M <= N.